plugins/Eq/EqFilter.h

```diff
@@ -136,6 +136,12 @@ class EqFilter : public StereoBiQuad
 		}
 	}
 
+	virtual void tick( sampleFrame *frame )
```

Comment:
What is the purpose of this function? Just seems like adding extra overhead... would probably be better to use inline functions for things such as this